graph/docu/docu_04graph_04.cc

```diff
@@ -39,8 +39,18 @@ SOFTWARE.
 TEST(Graph, GNUPlotScience) {
   // Where visualization meets science.
   using namespace bricks::gnuplot;
-const char* const extensions[2] = { "svg", "png" };
+#ifndef BRICKS_APPLE
+const char* const formats[2] = { "dumb", "pngcairo" };
```

Comment:
What is the difference between pngcairo and png?
What is bad about using png everywhere?